examples/portfolio/src/pages/projects.astro

```diff
@@ -20,7 +20,7 @@ const projects = (await Astro.glob("./project/**/*.md"))
 			s
 			most
 			recent
-			projects'
+			projects
```

Comment:
Ah this must have happened in #3862!

Would you mind updating the `description` attribute so that it says:

```
description="Learn about Jeanine White's most recent projects"
```

Reply:
Yes, introduced here: https://github.com/withastro/astro/pull/3862/files#diff-a739195a828a765d1e4de3a8e8bf4622ea9b80dc79ac379432843fcc5447a946R23

I've updated the description!

Reply:
Oops, totally missed this in my review of that PR, my apologies!